java/BUILD

```
@@ -20,6 +21,14 @@ DISTRO_VERSIONS = {
     "_debian10": versions_debian10,
 }
 
+[locale(
+    name = "locale_java" + distro_suffix,
+    deb = DISTRO_PACKAGES[distro_suffix]["libc-bin"],
+) for distro_suffix in [
+    "_debian9",
+    "_debian10",
+]]
```

Probably this will work:
```suggestion
) for distro_suffix in DISTRO_SUFFIXES]
```
Could you also update the code in line 35-38?